packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -139,7 +141,7 @@ describe('App: Settings', () => {
       cy.loginUser()
 
       cy.visitApp()
-      cy.findByText('Settings').click()
+      cy.findByTestId('sidebar-link-settings-page').click()
```

Comment:
Hmm, should this be `cy.get(SidebarSettingsLinkSelector)`? Or it's probably more typical for all the others to be `findByTestId('sidebar-link-settings-page')`, since the selector is a test id.